controllers/cronset_controller.go

```diff
@@ -49,12 +49,6 @@ func (r *CronSetReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	if err := ctrl.NewControllerManagedBy(mgr).
 		For(&batchv1alpha1.CronSet{}).
 		Owns(&batchv1.CronJob{}).
-		Complete(r); err != nil {
-		return err
-	}
-
-	if err := ctrl.NewControllerManagedBy(mgr).
-		For(&corev1.Node{}).
```

Comment:
Aha, we really didn't need to call NewControllerManagedBy separately and do `For(&corev1.Node{})`. I guess `Watches` registers the EventHandler for the Source(Node in our case) with the Informer.

https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/source@v0.14.1#Source.Start